components/post_view/post/post.jsx

```diff
@@ -195,6 +195,20 @@ export default class Post extends React.PureComponent {
 
     render() {
         const post = this.props.post || {};
+        const isEphemeral = Utils.isPostEphemeral(post);
+
+        // When user/s is added to channel by clicking @mention links,
+        // add-user-to-channel system message is posted at center while ephemeral message at RHS.
+        // This condition prevents add-user-to-channel ephemeral message at center posts.
+        if (
+            isEphemeral &&
+            post.root_id &&
+            post.props &&
```

Comment:
Maybe check the post.type instead of checking the props here? It seems a bit more obvious what you're looking for here

Reply:
Never mind. I misunderstood which message we were hiding